package.json

```diff
@@ -142,7 +143,7 @@
     "cordova-plugin-fcm-ng": "5.1.1",
     "cordova-plugin-file": "6.0.2",
     "cordova-plugin-globalization": "1.0.9",
-    "cordova-plugin-inappbrowser": "2.0.2",
+    "cordova-plugin-inappbrowser": "3.1.0",
```

Comment:
Maybe you're missing to add this version into `app-template` also

Reply:
good catch just updated